### DISCUSSION TITLE
Select item in a table, pass selection to another table

### DISCUSSION THREAD

**Discussion by suneworld** [EvotecIT/PSWriteHTML#458] [Q&A]
I would like to have a list (table) and when I click one of the computer names populate another section with data.
How do I do this?

**Comment by PrzemyslawKlys** [comment]
By another section do you mean another table? if so, that's doable using New-TableEvent where you populate data in two tables and then using new-tableevent match it up on some column which does search on it. Other ways are not possible atm
